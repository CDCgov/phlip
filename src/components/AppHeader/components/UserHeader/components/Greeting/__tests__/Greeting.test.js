import React from 'react'
import { shallow } from 'enzyme'
import { Greeting } from '../index'

const props = {
  firstName: 'Test',
  lastName: 'User',
  role: 'Admin'
}

describe('App Header - User Header - Greeting', () => {
  test('should render correctly', () => {
    expect(shallow(<Greeting {...props} />)).toMatchSnapshot()
  })
})