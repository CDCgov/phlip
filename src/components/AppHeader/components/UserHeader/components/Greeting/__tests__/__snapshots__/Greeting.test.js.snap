// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Header - User Header - Greeting should render correctly 1`] = `
<WithStyles(Typography)
  style={
    Object {
      "color": "white",
    }
  }
>
  Welcome, 
  Test
   
  User
  !
  <span
    style={
      Object {
        "fontStyle": "italic",
        "opacity": 0.6,
      }
    }
  >
      (
    Admin
    )
  </span>
</WithStyles(Typography)>
`;
